src/main/java/avenga/random/controller/DefaultController.java

```diff
@@ -0,0 +1,28 @@
+package avenga.random.controller;
+
+import avenga.random.service.SheetsService;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@CrossOrigin
```

Comment:
well, when we running js embedded there is no need in cross-origin if i am not mistaken